duckduckgo/iOS
# Improved bookmark import handling for DD html tags

Task/Issue URL: https://app.asana.com/0/414709148257752/1202348481335243/f
Tech Design URL:
CC:

**Description**:
This is a fix for the high bookmark import error rate that appears to be due to older Netscape / Firefox bookmark html files which have additional descriptive <DD> html tags which were causing imports to fail



**Steps to test this PR**:
1. Import a range of bookmark files especially ones with <DD> tags (there are multiple sample files within DuckDuckGoTests/MockFiles) and ensure they import successfully with no errors thrown



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
